public/Census_ACS_5yr/Census_ACS_5yr.py

```diff
@@ -10,22 +10,27 @@ def udf(
     
     from utils import acs_5yr_bounds
 
+    # convert bounds to tile
+    utils = fused.load("https://github.com/fusedio/udfs/tree/bb712a5/public/common/").utils
+    zoom = utils.estimate_zoom(bounds)
+    tile = utils.get_tiles(bounds, zoom=zoom)
+
     # different geometry details per zoom level
-    if bounds.z[0] > 12:
+    if tile.z[0] > 12:
```

Comment:
```suggestion
    if zoom > 12:
```

We estimated the zoom first, so can use that variable directly, I think?

Reply:
Yup!